### PR TITLE
Modify the value of result.ref in the integration test to eliminate some of the warning in github

### PR DESCRIPTION
**Pull request by Zhuxuegang2022**
### Reminder
- [x] Have you linked an issue with this pull request?
- [x] Have you noticed possible changes of behavior below or in the linked issue?
- [ ] Have you explained the changes of codes in core modules of ESolver, HSolver, ElecState, Hamilt, Operator or Psi? (ignore if not applicable)

### Linked Issue
Related to issues of #3242 and #2844 and PR #3382

### What's changed?
I modify the value of result.ref in the integration test, which is used to delete some of the warning in github, 
It should be note that some other warning should pay more attention, because the value get from my bohrium computer is different to github, the related test are given in the issues of #3414

### Any changes of core modules? (ignore if not applicable)
no

### CODE REVIEW

**Comment by Zhuxuegang2022** [comment]
There are 8 warnings, which is special when I change the result.ref.
101_PW_upf201_uspp_NaCl
104_PW_NC_magnetic
109_PW_CR_fix_a
109_PW_CR_fix_ac
116_PW_scan_Si2_nspin2
281_NO_KP_HSE
286_NO_KP_CR_HSE
601_NO_TDDFT_CO
______
Between them the modified by me is
102_PW_BPCG, which need pay more attention later to get the reason under it
____
the warning that change by other PR #3382  is also changed in this PR
104_PW_NC_magnetic
109_PW_CR_fix_a
109_PW_CR_fix_ac
116_PW_scan_Si2_nspin2
____
the warning that I  mentiond one case in the issue #3414, but when compile the software by some other libiray it seem to get accurate value
101_PW_upf201_uspp_NaCl
281_NO_KP_HSE
286_NO_KP_CR_HSE

File: tests/integrate/101_PW_15_paw/result.ref [hunk 1]
```diff
@@ -1,7 +1,7 @@
-etotref -200.9137994781310681
-etotperatomref -100.4568997391
-totalforceref 0.126270
+etotref -200.9130495851335070
+etotperatomref -100.4565247926
+totalforceref 3.246042
```
Comment (dyzheng) [inline tests/integrate/101_PW_15_paw/result.ref:3]: @wenfei-li The change of reference looks so large, can you check this case for current version?

Comment (wenfei-li) [reply tests/integrate/101_PW_15_paw/result.ref:3]: I checked and the new results are correct

**Comment by Zhuxuegang2022** [comment]
385_NO_GO_RE_S1_HSE/result.ref
386_NO_GO_MD_S1_HSE/result.ref
while I have rebuld abacus with enable paw, but the HSE could not calculate, so the time of former tow test not change to correct.

**Comment by Zhuxuegang2022** [comment]
inorder to persorm hse, I need build the abacus with more four lib:ENABLE_LIBRI  ENABLE_LIBCOMM  ENABLE_PAW ENABLE_LIBXC.

**Comment by Zhuxuegang2022** [comment]
What more when I change a computer, the warning value of tree warning get the same value with github:"109_PW_CR_fix_a 109_PW_CR_fix_ac 116_PW_scan_Si2_nspin2"

**Comment by mohanchen** [comment]
The PR lasts too long, I will close it. We still welcome new PRs.

**Pull request closed without merging**